environments/ithc/backend_lb_config.yaml

```diff
@@ -130,11 +130,9 @@ gateways:
     # SIDAM
       - product: idam
         component: api
-        cookie_based_affinity: Enabled
         ssl_enabled: true
       - product: idam
         component: web-admin
-        cookie_based_affinity: Enabled
```

Comment:
We need to preserve the affinity for web-admin

Reply:
why's that?

Reply:
Admin needs it because of the authentication cookie. We are planning to fix this with redis and idam-web-admin refactoring. It needs a lot but has never been prioritised. 
When you log in web-admin keeps a cookie reference in your browser a reference to your session which is just a key - value map in memory. 
Only one web-admin has your session in memory so you can redirect and get unauthorised errors on every click.